scripts/start

```diff
@@ -4,6 +4,14 @@
 
 const app = require("../src/app");
 const http = require("http");
+const db = require("../src/utils/dbHelper");
+const commands = require("../src/utils/constants").COMMANDS;
+
+// Create custom commands for emoji logging
+commands.forEach(({ name, emoji }) => console[name] = (...args) => console.log(emoji + ' ' + args.join(', ')));
+
+const dbConnect = new db.Database();
+console.tick(" Database connection successful");
```

Comment:
The extra whitespace here is not required:
```suggestion
console.tick("Database connection successful");
```
Since you're already appending a whitespace after the emoji in line 11.